scene/gui/line_edit.cpp

```diff
@@ -2629,6 +2648,16 @@ bool LineEdit::is_emoji_menu_enabled() const {
 	return emoji_menu_enabled;
 }
 
+void LineEdit::set_backspace_deletes_composite_character_enabled(bool p_enabled) {
+	if (backspace_deletes_composite_character_enabled != p_enabled) {
+		backspace_deletes_composite_character_enabled = p_enabled;
+	}
```

Comment:
```suggestion
	backspace_deletes_composite_character_enabled = p_enabled;
```
It's not doing anything extra, so no need for the check.